src/main/java/org/team1540/robot2018/Robot.java

```diff
@@ -376,4 +204,199 @@ public void teleopPeriodic() {
+        }),
+        new DecisionNode(Posession.NO_DATA, MotionProfile.GO_STRAIGHT).setMessage(new Message("Could not get match data, reverting to base auto", true))
+    })),
+    LEFT_SCALE_THEN_SWITCH("Left Scale Then Switch", new DecisionNode(new DecisionNode[]{
+        new DecisionNode(Posession.SCALE_OWNED_LEFT, MotionProfile.LEFT_SCALE),
+        new DecisionNode(Posession.SCALE_OWNED_RIGHT, new DecisionNode[]{
+            new DecisionNode(Posession.SWITCH_OWNED_LEFT, MotionProfile.LEFT_HOOK),
+            new DecisionNode(Posession.SWITCH_OWNED_RIGHT, MotionProfile.GO_STRAIGHT)
+        }),
+        new DecisionNode(Posession.NO_DATA, MotionProfile.GO_STRAIGHT).setMessage(new Message("Could not get match data, reverting to base auto", true))
+    })),
+    LEFT_SCALE_NO_SWTICH("Left Scale No Switch", new DecisionNode(new DecisionNode[]{
+        new DecisionNode(Posession.SCALE_OWNED_LEFT, MotionProfile.LEFT_SCALE),
+        new DecisionNode(Posession.SCALE_OWNED_RIGHT, MotionProfile.GO_STRAIGHT),
+        new DecisionNode(Posession.NO_DATA, MotionProfile.GO_STRAIGHT).setMessage(new Message("Could not get match data, reverting to base auto", true))
+    })),
+    LEFT_DOUBLE_SCALE_NO_SWITCH("Left Double Scale No Switch", new DecisionNode(new DecisionNode[]{
+        new DecisionNode(Posession.SCALE_OWNED_LEFT, MotionProfile.LEFT_SCALE_NO_SWITCH),
+        new DecisionNode(Posession.SCALE_OWNED_RIGHT, MotionProfile.GO_STRAIGHT),
+        new DecisionNode(Posession.NO_DATA, MotionProfile.GO_STRAIGHT).setMessage(new Message("Could not get match data, reverting to base auto", true))
+    })),
+    LEFT_HOOK_SWITCH_THEN_DOUBLE_SCALE("Left Hook Switch Then Double Scale", new DecisionNode(new DecisionNode[]{
+        new DecisionNode(Posession.SWITCH_OWNED_LEFT, MotionProfile.LEFT_HOOK),
+        new DecisionNode(Posession.SCALE_OWNED_LEFT, MotionProfile.LEFT_SCALE_STRAIGHT),
+        new DecisionNode(Posession.NO_DATA, MotionProfile.GO_STRAIGHT).setMessage(new Message("Could not get match data, reverting to base auto", true))
+    })),
+    LEFT_HOOK_SWITCH_THEN_SCALE("Left Hook Switch Then Scale", new DecisionNode(new DecisionNode[]{
+        new DecisionNode(Posession.SWITCH_OWNED_LEFT, MotionProfile.LEFT_HOOK),
+        new DecisionNode(Posession.SCALE_OWNED_LEFT, MotionProfile.LEFT_SCALE_STRAIGHT),
+        new DecisionNode(Posession.SCALE_OWNED_RIGHT, MotionProfile.GO_STRAIGHT),
+        new DecisionNode(Posession.NO_DATA, MotionProfile.GO_STRAIGHT).setMessage(new Message("Could not get match data, reverting to base auto", true))
+    })),
+    MIDDLE("Middle", new DecisionNode(new DecisionNode[]{
+        new DecisionNode(Posession.SWITCH_OWNED_LEFT, MotionProfile.MIDDLE_TO_LEFT_SWITCH),
+        new DecisionNode(Posession.SWITCH_OWNED_RIGHT, MotionProfile.MIDDLE_TO_RIGHT_SWITCH),
+        new DecisionNode(Posession.NO_DATA, MotionProfile.GO_STRAIGHT).setMessage(new Message("Could not get match data, reverting to base auto", true))
```

Comment:
maybe just make this the default occurrence?

Reply:
No, a bunch of the ones you wrote don't include it.

Reply:
Also better flexibility.

Reply:
They should have, I was just being lazy. Note that if you _do_ make this the default occurrence you need a "do nothing" provision.

Reply:
eh i'll stick with it for flexibility, it's fine